include/deal.II/fe/fe_enriched.h

```diff
@@ -251,15 +251,15 @@ class FE_Enriched : public FiniteElement<dim, spacedim>
    * define dummy functions. Below is an example which uses two functions with
    * the first element to be enriched and a single function with the second one.
    * @code
-   * FE_Enriched<dim> fe
-   * (&fe_base,
-   * {&fe_1, &fe_2},
-   * {{[=] (const typename Triangulation<dim>::cell_iterator &) -> const
-   * Function<dim> * {return &fe_1_function1;},
-   *   [=] (const typename Triangulation<dim>::cell_iterator &) -> const
-   * Function<dim> * {return &fe_1_function2;}},
-   *  {[=] (const typename Triangulation<dim>::cell_iterator &) -> const
-   * Function<dim> * {return &fe_2_function;}}});
+   * FE_Enriched<dim> fe(
+   *   &fe_base,
+   *   {&fe_1, &fe_2},
+   *   {{[=] (const typename Triangulation<dim>::cell_iterator &)
+   *       -> const Function<dim> * {return &fe_1_function1;},
```

Comment:
Can you break the body of the lambda onto a line of its own? Or does reflow-comments destroy that again? 

Same below of course.

Reply:
ReflowComments will only touch lines which are longer than 80 characters. I can do that change.

Reply:
Thanks. The existing style look too much like we're multiplying `Function<dim>` by the thing in braces :-)